### DISCUSSION TITLE
Please review some additions I've made to the manual

### DISCUSSION THREAD

**Discussion by jpellegrini** [xournalpp/xournalpp#3227] [General]
Hello!
I have added some entries to the manual in the wiki (hand tool, selection tool, snapping and help).  Are those ok?

**Comment by rolandlo** [comment]
Hi @jpellegrini 
Thanks for contributing. Your additions seem mostly fine to me. Some minor things I have spotted:
- the section about the hand tool is not quite accurate when you have multiple pages. As long as not all pages are in view, the hand tool will work.
- Could you add degree signs for the angles?
- I think "Grid Snapping" should be with with capitol "G" in `Edit > grid Snapping`

Consider contributing to the [website](https://xournalpp.github.io) as well. That will mostly replace/subsume the User Manual.

**Reply by jpellegrini** [reply]
Hi @rolandlo !
Thanks, I'll update the entries I've made and will check how I can contribute to the website. Thanks for replying!
